### PR TITLE
feat(core): add language param to createSponsoredMember and createSpo…

### PR DESCRIPTION
**Pull request by xflord**
…nsoredMembersFromCSV

* added language parameter that decides in what language to send email notification to the mentioned methods

### CODE REVIEW

File: perun-openapi/openapi.yml [hunk 1]
```diff
@@ -10339,6 +10340,7 @@ paths:
                 namespace: { type: string }
                 validityTo: { type: string }
                 sendActivationLinks: { type: boolean }
+                language: { type: string }
```
Comment (Johaney-s) [inline perun-openapi/openapi.yml:10343]: i'd say lets add the parameter also to `createSponsoredMembers` method

File: perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java [hunk 2]
```diff
@@ -176,6 +176,10 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
 		@Override
 		public RichMember call(ApiCaller ac, Deserializer params) throws PerunException {
```
Comment (Johaney-s) [inline perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java:177]: please update javadocs with new parameter

**Review by Johaney-s — COMMENTED** [review]
please write a note for next release deployment that more language options are now enabled so templates should be checked/created for them via the entityless attributes.

**Comment by xflord** [comment]
> please write a note for next release deployment that more language options are now enabled so templates should be checked/created for them via the entityless attributes.

Did so over here [](https://perunaai.atlassian.net/wiki/spaces/STRIBOG/pages/99287047/Sprint+50 ), please make sure that what I wrote makes sense.

**Review by Johaney-s — APPROVED** [review]

**Review by mattjoke — APPROVED** [review]

**Pull request merged**